src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/StreamingClientManager.java

```diff
@@ -72,25 +83,46 @@ public void createAllStreamingClients(
     this.maxTasks = maxTasks;
 
     int clientCount = (int) Math.ceil((double) maxTasks / (double) numTasksPerClient);
-    LOGGER.info(
-        "Creating {} clients for {} tasks with max {} tasks per client",
-        clientCount,
-        maxTasks,
-        numTasksPerClient);
 
     Properties clientProperties = new Properties();
     clientProperties.putAll(
         StreamingUtils.convertConfigForStreamingClient(new HashMap<>(connectorConfig)));
 
+    // Override only if bdec version is explicitly set in config, default to the version set inside
+    // Ingest SDK
+    Map<String, Object> parameterOverrides = new HashMap<>();
+    // default bdec version is THREE defined in Ingest SDK dependency [THREE corresponds to PARQUET]
```

Comment:
I suggest we remove this comment because the default could change later

Reply:
Ack .